modules/mbedtls/crypto_mbedtls.cpp

```diff
@@ -186,6 +188,78 @@ Error X509CertificateMbedTLS::save(String p_path) {
 	return OK;
 }
 
+bool HMACContextMbedTLS::is_md_type_allowed(mbedtls_md_type_t p_md_type) {
+	mbedtls_md_type_t allowed[] = { MBEDTLS_MD_SHA1, MBEDTLS_MD_SHA256 };
+
+	for (size_t i = 0; i < sizeof(allowed); i++) {
```

Comment:
this looks like an overkill.
can you use a switch/case?

Reply:
I could use switch, sure.